app/pom.xml

```diff
@@ -8,7 +8,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>3.0.2</version>
+        <version>3.1.0</version>
```

Comment:
There is already 3.1.1, let's use the latest and greatest.

Reply:
k making the changes

Reply:
Made the changes can you review the PR once